templates/admin/job_template/index.html.ep

```diff
@@ -15,12 +15,20 @@
         false
     % }
     ;
+    % if ($group->template) {
+    toggleTemplateEditor();
```

Comment:
Indentation looks wrong.

Reply:
My editor seems to get confused with the mix... should be corrected now, also another instance at the bottom with HTML